rules/sinks/storages/facebooklocalcache/java.yaml

```diff
@@ -0,0 +1,14 @@
+sinks:
+  - id: Storages.Facebook.LocalCache
+    name: Facebook Android Local Cache
+    patterns:
+      - "(?i)(.*)(com.facebook.Session.saveTokenToCache:void(com.facebook.AccessToken))(.*)"
+      - "(?i)(.*)(com.facebook.TestSession.setTokenInfo:void(com.facebook.AccessToken))(.*)"
```

Comment:
This looks like some testing package, if yes we can remove this from sink

Reply:
@gaurav-gogia I guess we can have the rule little open 
Ex - `com.facebook.Session.saveTokenToCache.*`

So that we don't have to end up writing rule for each variation of method